am-lib/src/main/java/uk/gov/hmcts/reform/amlib/internal/repositories/AccessManagementRepository.java

```diff
@@ -53,4 +55,8 @@
 
     @SqlQuery("select access_management_type from roles where role_name = :roleName")
     AccessType getRoleAccessType(String roleName);
+
+    @SqlQuery("select service_name, resource_type, resource_name from default_permissions_for_roles where role_name = :roleName and default_permissions_for_roles.permissions = 1")
```

Comment:
No need to prefix column name with table name if there is no ambiguity. 

```suggestion
    @SqlQuery("select service_name, resource_type, resource_name from default_permissions_for_roles where role_name = :roleName and permissions = 1")
```